SimplenoteWidgets/Views/NoteListTable.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftUI
+
+struct NoteListTable: View {
+    let entry: ListWidgetEntry
+    let numberOfRows: Int
+    let geometry: GeometryProxy
+
+    var body: some View {
+        let width = geometry.size.width - Constants.sidePadding
+        let height = round((geometry.size.height - Constants.headerSize) / CGFloat(numberOfRows))
+        let proxies = entry.noteProxys
+
+        ForEach(.zero ..< numberOfRows) { index in
```

Comment:
It is setup this way because there might be fewer notes than the max for the widget.  By specifying the number of rows expected we can continue to generate rows even if there are no notes